packages/cli/src/databases/repositories/execution.repository.ts

```diff
@@ -294,6 +294,16 @@ export class ExecutionRepository extends Repository<ExecutionEntity> {
 		);
 	}
 
+	async findAllPotentiallyRunningExecutionIds() {
+		const executions = await this.find({
+			where: {
+				status: In(['running', 'new', 'unknown']),
```

Comment:
Why not `waiting` also? For reference, pruning defines unfinished executions like this: `Not(In(['new', 'running', 'waiting']))`

Reply:
we need to find those that are in a 'running' sort of state during startup, so we can set them to crashed (since after the main instance crashed, they can't be running any more)